Commands/timeout.py

```
@@ -1,84 +1,65 @@
+    channel_id = message["tags"]["room-id"]
+    timeout_id = get_user_id(user_to_timeout)
+    moderator_id = get_user_id('bluepagmanbot')
+
+    # Attempt timeout
+    result = timeout(channel_id, moderator_id, timeout_id)
+
+    if result == "Success":
+        self.users.update_one({'user': cmd.nick}, {'$inc': {'timeout': -1}})
+    else:
+        responses = {
+            "Bad": f"Timeout failed, either the user is a mod/broadcaster or the user does not exist.",
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)